mParticle-Apple-SDK/Include/MPKitProtocol.h

```diff
@@ -143,7 +143,8 @@
                                           config:(MPRoktConfig * _Nullable)config
                                        callbacks:(MPRoktEventCallback * _Nullable)callbacks
                                     filteredUser:(FilteredMParticleUser * _Nonnull)filteredUser;
-
+- (nonnull MPKitExecStatus *)setWrapperSdk:(MPWrapperSdk)wrapperSdk
+                                   version:(nonnull NSString *)wrapperSdkVersion;
```

Comment:
nit -  colon alignment

Reply:
Nevermind. Was skimming and realized this is a new method and not adding arguments to the method above it 😭